### ISSUE TITLE
Unknown OCaml version 4.13.0+dev0-2020-10-19

### ISSUE THREAD

**Issue by shakthimaan** [let-def/migrate-parsetree#111]
I am trying to build `ocaml-migrate-parsetree-2.1.0` with upstream OCaml (https://github.com/ocaml/ocaml/archive/trunk.tar.gz) for https://github.com/ocaml-bench/sandmark, and it fails with:
```
 ocaml src/ast-version (exit 1)
(cd _build/default/src && /home/guest/testing/sandmark/_opam/4.13.0+trunk/bin/ocaml config/gen.ml 4.13.0+dev0-2020-10-19)
Unknown OCaml version 4.13.0+dev0-2020-10-19
```
It builds fine for https://github.com/ocaml/ocaml/archive/4.12.0.tar.gz. 
How can this be fixed?

**Comment by kit-ty-kate** [comment]
This was fixed in https://github.com/ocaml-ppx/ocaml-migrate-parsetree/pull/114

**Comment by pitag-ha** [comment]
Since this was fixed when adding the 4.13 support, I'm going to close the issue.
